src/main/app/forms/models/fileTypes.ts

```diff
@@ -44,4 +48,22 @@ export class FileTypes {
     })
     return acceptedFileExtensions
   }
+
+  static isOfAcceptedMimeType (filePath: string): Promise<boolean> {
+    return new Promise(accept => {
+      const buffer = readChunk.sync(filePath, 0, 4100)
+      if (fileType(buffer) !== null && fileType(buffer).mime === 'application/x-msi') {
```

Comment:
Why `x-msi`? MSI is a Windows installer package, isn't it?

Reply:
Also, what do you think about extracting the content of this `if` into a function named meaningfully, e.g. `detectMicrosoftDocumentType`?

Reply:
the package we are using have lumped all these old type of files as ```x-msi``` and then you have to look into the file to see which type of windows file it is. Sure I can extract it out to another function